examples/OneOpenAir/OneOpenAir.ino

```diff
@@ -1053,13 +1057,26 @@ static void updatePm(void) {
                     ag->pms5003t_1.temperatureCompensated(measurements.temp_1));
       Serial.printf("[1] Relative Humidity compensated: %f\r\n",
                     ag->pms5003t_1.humidityCompensated(measurements.hum_1));
+
+      ag->pms5003t_1.resetFailCount();
     } else {
-      measurements.pm01_1 = utils::getInvalidPMS();
-      measurements.pm25_1 = utils::getInvalidPMS();
-      measurements.pm10_1 = utils::getInvalidPMS();
-      measurements.pm03PCount_1 = utils::getInvalidPMS();
-      measurements.temp_1 = utils::getInvalidTemperature();
-      measurements.hum_1 = utils::getInvalidHumidity();
+      if (configuration.hasSensorPMS1) {
+        ag->pms5003t_1.updateFailCount();
+        Serial.printf("[1] PMS read failed: %d\r\n", ag->pms5003t_1.getFailCount());
```

Comment:
The message is misleading, the read hasn't failed here?

Reply:
it's failed if `configuration.hasSensorPMS1`=`true`